src/lib/containers/personal_access_token/AccessTokenCard.tsx

```diff
@@ -26,12 +26,6 @@ export const AccessTokenCard: React.FunctionComponent<AccessTokenCardProps> = ({
 
   const isExpired = accessToken.state === 'EXPIRED'
 
-  useEffect(() => {
-    // For reasons unknown, the tooltips in this component would not load in Synapse.org without this
-    // see https://github.com/wwayne/react-tooltip/issues/344
-    ReactTooltip.rebuild()
```

Comment:
Anyone else worried about the integration?
Could you create the umd build locally with these changes, and drop them into a running SWC client to get a preview of the integration (see if the tooltips are still working)?  Let me know if you need help with this, and I can show how to do this from my machine